### DISCUSSION TITLE
Are there any suggestions for typography settings?

### DISCUSSION THREAD

**Discussion by ivanhertanto** [thuliteio/doks#1313] [Ideas]
It would be great to have Typography Settings available while reading DOKS, as it would eliminate the need to zoom in and out in the browser.

Typography Settings (font size, font family, line height by reader)

**Comment by h-enk** [comment]
Interesting idea. This will not directly be picked up, but it is noted!

Something like implemented in [Simply Cyan](https://thulite.io/themes/simply-cyan/)

**Reply by ivanhertanto** [reply]
Thank you for your response.

Just for a reference, typography tools similar to those found on webnovel.com or scribblehub.com may be of interest.

![2024-12-23_18-29](https://github.com/user-attachments/assets/e140b005-cd5f-43e2-ba18-7c8f6004e575)
![2024-12-23_18-29_1](https://github.com/user-attachments/assets/a6d67f08-3d12-487c-bb65-d6f951ce570b)

**Reply by h-enk** [reply]
Thanks for sharing!
